src/multirust

```diff
@@ -1341,6 +1352,18 @@ show_override() {
     fi
 }
 
+show_toolchain() {
+    local _toolchain=
+    if find_override; then
+        _toolchain="$RETVAL_TOOLCHAIN"
+    elif find_default; then
+        _toolchain="$RETVAL_TOOLCHAIN"
+    else
+        _toolchain="No default"
```

Comment:
Should it print something if it can't find a default, or should it print nothing?